src/pages/documentation/adobe-campaign-classic/api-reference.md

```diff
@@ -30,6 +30,20 @@ iOS
 
 <Tabs query="platform=ios&api=register-device"/>
 
+## registerExtension
+
+This `registerExtension` API registers the Adobe Campaign Classic extension with the Mobile Core.
```

Comment:
"The `registerExtension`..."

Reply:
I noticed it too but got distracted with something else before fixing it. Thanks.